docker/bbp-network/start_all.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
```

Comment:
very cool.
 wonder if you can incorporate register-dids in here, maybe with an env var or param to call it or not